### PR TITLE
add CITATION.cff

### PR DESCRIPTION
**Pull request by AnnaSchwerdtel**
Adding a CITATION.cff file for future users to reference the repository. Please check the file for accuracy, as I was not part of the developement process and may not have had the correct information avaliable to me. 
Please especially check for authorship credits, I could only find sufficient information on one of the authors through GitHub, even though more than one person was involved in the developement.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: unknown data framework
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
```
Comment (greenBene) [inline CITATION.cff:10]: The following authors are missing:  
* Lu Gan (https://www.gesis.org/en/institute/about-us/staff/person/Lu.Gan)
* Sebastian Tiesler (https://www.ibi.hu-berlin.de/de/institut/personen/tiesler/)
* Yousef Younes (https://orcid.org/0000-0003-1271-3633)

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - given-names: Benedikt Maria
+    family-names: Beckermann
+    affiliation: Schloss Dagstuhl Leibniz-Zentrum für Informatik
```
Comment (greenBene) [inline CITATION.cff:13]: Please format the affiliation as `Schloss Dagstuhl - Leibniz Center for Informatics`

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    orcid: 'https://orcid.org/0009-0008-3920-6109'
+    email: hello@benedikt.click
```
Comment (greenBene) [inline CITATION.cff:15]: Please use my work email here :) `benedikt.beckermann@dagstuhl.de`

**Review by greenBene — CHANGES_REQUESTED** [review]
And again thank you! ^^ Here three authors are missing! :)

File: CITATION.cff (hunk 1 above)
Comment (AnnaSchwerdtel) [reply CITATION.cff:10]: Thank you for the quick reply and for the information about the missing authors :)

**Pull request merged**